docs/magento2/detect-admin-users.mdx

```diff
@@ -238,7 +238,7 @@ without worrying about session cookies issues.
 Here is a sample `curl` request to achieve this:
```

Comment:
Maybe highlight the need to escape the JSON body ? 

```suggestion
Here is a sample `curl` request to achieve this (note the escaped JSON data) :
```

Reply:
Please have a look at the video in the description, I split the two commands up, as @real34 showed me this morning that it can simply be run in the browser, that's why initially it was not encoded.